cdap-docs/examples-manual/source/examples/data-cleansing.rst

```diff
@@ -30,7 +30,7 @@ of the application are tied together by the class ``DataCleansing``:
 
 .. literalinclude:: /../../../cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java
    :language: java
-   :lines: 32-
+   :lines: 30-
```

Comment:
Can you leave out this change? It was due to the change to DataCleansing.java, as in: https://github.com/caskdata/cdap/pull/8858.
I had reverted the change to DataCleansing.java, but am bringing it back in that PR.